packages/styles/stepper.css

```
@@ -72,8 +75,13 @@
   box-shadow: 0 0 0 2px var(--accent-dark);
 }
 
+.Stepper__step--current .Stepper__step-indicator::before {
```

Nit: we typically use `:before` anywhere we set pseudo selectors, so this is mostly for consistency purposes.

```suggestion
.Stepper__step--current .Stepper__step-indicator:before {
```